src/github/push.ts

```diff
@@ -22,14 +23,19 @@ export default async (context: Context, jiraClient): Promise<void> => {
 		installation: context.payload?.installation
 	};
 
-	if (payload.commits?.length === 0) {
+	if (!payload.commits?.length) {
 		context.log(
 			{ noop: "no_commits" },
 			"Halting further execution for push since no commits were found for the payload"
 		);
 		return;
 	}
 
+	// If there's less than 20 commits (the number of commits the github API returns per call), just process it immediately
+	if(payload.commits?.length < 20 && await booleanFlag(BooleanFlags.PROCESS_PUSHES_IMMEDIATELY, true)) {
```

Comment:
Please set the default value to `false`, which is the current state. We want to explicitly set it to `true` via the feature flag.

Reply:
we can keep it like this, since LaunchDarkly will resolve the value to `false` anyways, as long as the Feature Flag isn't activated, yet